src/NavList/NavList.tsx

```diff
@@ -235,18 +236,18 @@ Divider.displayName = 'NavList.Divider'
 // ----------------------------------------------------------------------------
 // NavList.Group
 
-export type NavListGroupProps = {
+export type NavListGroupProps = Omit<ActionListGroupProps, 'selectionVariant' | 'variant' | 'auxiliaryText'> & {
   children: React.ReactNode
-  title?: string
 } & SxProp
 
+const defaultSx = {}
 // TODO: ref prop
-const Group = ({title, children, sx: sxProp = {}}: NavListGroupProps) => {
+const Group: React.VFC<NavListGroupProps> = ({title, children, sx: sxProp = defaultSx, ...props}) => {
```

Comment:
`VFC` is `FC` but `children` is not implied, which makes it much easier to transition to react 18, where `FC` doesn't have children by default - since it ensures we add them manually